boards/arm/cc1352p1_launchxl/boosterpack_connector.dtsi

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
if this is unchanged from `cc1352r_launchxl`, then just use a relative include directory. Same thing was done for the `cc1342r_sensortag`.
```
#include "../cc1352r1_launchxl/boosterpack_connector.dtsi"
```